modules/volume/shaders/helper.glsl

```diff
@@ -22,21 +22,22 @@
  * OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                                         *
  ****************************************************************************************/
 
-#define KAMELEON_PI      3.14159265358979323846  /* pi */
-#define KAMELEON_SQRT1_3 0.57735026919           /* 1/sqrt(3) */
+#define VOLUME_PI      3.14159265358979323846  /* pi */
```

Comment:
Can replace this with a  `glm::pi<double>()` call from GLM header `<glm/gtc/constants.hpp>`

Same for the following line.  By specifying the variables as `constexpr`, you can do the computation at compile time and write:  `constexpr double sqrt1_3 = 1.0 / sqrt(3);`